src/components/fields/ArrayField.js

```diff
@@ -23,6 +23,14 @@ function ArrayFieldTitle({TitleField, idSchema, title, required}) {
   return <TitleField id={id} title={title} required={required} />;
 }
 
+function DescriptionFieldTitle({DescriptionField, idSchema, description}) {
```

Comment:
`DescriptionFieldTitle` sounds odd; could we come with a slightly better name? *Wrapper? *Container?